pkg/operators/ebpf/converters.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/datasource"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/utils/annotations"
+)
+
+const (
+	enumTargetNameAnnotation = "ebpf.formatter.enum"
```

Comment:
nit:
```suggestion

const (
	enumTargetNameAnnotation = "ebpf.converter.enum"
```

Or maybe rename the file.

Reply:
Renamed the file to have a single formatter concept.